net-wireless/wireless-regdb/wireless-regdb-20220812.ebuild

```
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="wireless-regdb-${PV:0:4}.${PV:4:2}.${PV:6:2}"
+DESCRIPTION="Binary regulatory database for CRDA"
+HOMEPAGE="https://wireless.wiki.kernel.org/en/developers/regulatory/wireless-regdb"
+SRC_URI="https://www.kernel.org/pub/software/network/${PN}/${MY_P}.tar.xz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+src_compile() {
+	einfo "Recompiling regulatory.bin from db.txt would break CRDA verify. Installing unmodified binary version."
```

This is pretty silly, tbh. We should really have it optional with some masked flag.

Also:
```suggestion
	einfo "Recompiling regulatory.bin from db.txt would break CRDA verification. Installing unmodified binary version."
```